functests/install-script.sh

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env bash
+
+# This file is part of MinIO DirectPV
+# Copyright (c) 2022 MinIO, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+echo "Installing direct PV"
+
+operating_system=""
+darwin="Darwin"
+linux="Linux"
+windows="Windows"
+
+function directpv(){
+    read operating_system kubectl_url minikube_url < <(initialize)
+    verify_kubectl
+    verify_krew 
+    kubectl krew install directpv
+    kubectl directpv install
```

Comment:
To install `directpv`, we need `KUBECONFIG` set by the user to install on decided k8s for the user.